src/components/org/PositionTimeline/components/TimelineSlider/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { formatDate } from '@equinor/fusion';
```

Comment:
Wasn't this file added in the TimlineComponent pr?

Reply:
This was a duplicate, and is deleted.